projects/plugins/boost/app/modules/optimizations/page-cache/Page_Cache_Setup.php

```diff
@@ -186,6 +186,17 @@ public static function deactivate() {
 		return true;
 	}
 
+	/**
+	 * Creates the advanced-cache.php file and adds the WP_CACHE define to wp-config.php
+	 * Fired when the plugin is activated.
+	 */
+	public static function activate() {
+		if ( Boost_Cache_Settings::get_instance()->get_enabled() ) {
+			self::run_setup();
+		}
```

Comment:
I think this should probably be in `class-jetpack-boost.php` and make sure that `run_setup` only runs if the Module is active AND boost cache is supposed to be active.

Reply:
Code has been updated!